src/index.ts

```diff
@@ -150,47 +150,161 @@ const extension: JupyterFrontEndPlugin<void> = {
       }
     });
 
+    window.onerror = function(msg, url, lineNo, columnNo, error): boolean {
```

Comment:
Maybe we want to make this function an arrow function to keep the same style for all hooks?